app/models/cloud_port.rb

```diff
@@ -0,0 +1,20 @@
+class CloudPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
+  belongs_to :device, :polymorphic => true
+
+  has_and_belongs_to_many :security_groups
+
+  has_many :floating_ips
+
+  serialize :allowed_address_pairs
+  serialize :extra_dhcp_opts
```

Comment:
I am using naming that is used and documented on OpenStack side. Is it better to change it?